Description: Remove CMakeLists.txt from external project
 This file is breaking the build as it contains refernces to
 components outside the unified runtime tree.
Author: Will French <will.french@canonical.com>
Last-Update: 2025-07-30
---
--- unified-runtime-0.11.8.orig/cmake/helpers.cmake
+++ unified-runtime-0.11.8/cmake/helpers.cmake
@@ -230,6 +230,8 @@ function(FetchSource GIT_REPOSITORY GIT_
         WORKING_DIRECTORY ${DEST})
     execute_process(COMMAND git checkout --quiet ${GIT_TAG}
         WORKING_DIRECTORY ${DEST})
+    execute_process(COMMAND rm ${GIT_DIR}/CMakeLists.txt
+        WORKING_DIRECTORY ${DEST})
 endfunction()
 
 # A wrapper around FetchContent_Declare that supports git sparse checkout.
